src/TestSuite/Fixture/FixtureInjector.php

```diff
@@ -14,6 +14,21 @@
  */
 namespace Cake\TestSuite\Fixture;
 
+if (class_exists('PHPUnit_Runner_Version')) {
+    if (version_compare(\PHPUnit_Runner_Version::id(), '5.7', '<')) {
+        trigger_error(sprintf('Your PHPUnit Version must be at least 5.7.0 to use CakePHP Testsuite, found %s', \PHPUnit_Runner_Version::id()), E_USER_ERROR);
+    }
+    class_alias('PHPUnit_Framework_Test', 'PHPUnit\Framework\Test');
+    class_alias('PHPUnit_Framework_Warning', 'PHPUnit\Framework\Warning');
```

Comment:
@markstory my reasoning was the following:
If a old PHPUNIT version class is found, I need de define aliases
If I need to define the same aliases in several places, I check if it was not done already.